modules/workload-identity/scripts/kubectl_wrapper.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
Should we move this into a common location? Maybe the gcloud module?

Reply:
I think thats a great idea, considering we now have copies here, in the crd module, asm module etc. Do you mean as a gcloud submodule or did you have another approach? Also might it might better to do it alongside #404

Reply:
Yeah, maybe a `gcloud` submodule?